lib/cryptoexchange/exchanges/dydx_perpetual/services/pairs.rb

```diff
@@ -17,6 +17,8 @@ def adapt(output)
             Cryptoexchange::Models::MarketPair.new(
               base:   base,
```

Comment:
I think the base should be `BTC` instead of `PBTC` because in trade page base is btc, the api use `PBTC-USDC` because this is perpatual bitcoin. If use PBTC in base means this is refer to other coin

Reply:
👍